raster/src/main/scala/geotrellis/raster/sigmoidal/SigmoidalContrast.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2016 Azavea.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.raster.sigmoidal
+
+import geotrellis.raster._
+import geotrellis.raster.histogram._
+
+
+object SigmoidalContrast {
+
+  private def _T(
+    cellType: CellType, alpha: Double, beta: Double
+  )(_u: Double): Double = {
```

Comment:
using an underscore in a param name is out of normal scala style - is there reason to use it? Could we replace this with a longer, more descriptive name instead?